### PR TITLE
Updating chatmessage template after login PR merge

### PR DESCRIPTION
**Pull request by patrisiakaplun**
updating message.html, test_chatmessage.py, urls.py and views.py

image

image

### CODE REVIEW

File: chatmessage/test_chatmessage.py [hunk 1]
```diff
@@ -5,26 +5,24 @@
 
 @pytest.mark.django_db
 def test_get_chat(client, professional, demo_client):
```
Comment (guybeckenstein) [inline chatmessage/test_chatmessage.py:7]: please use make_client and not demo_client

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by idosinger98 — COMMENTED** [review]
Please show how it looks on the professional side

**Comment by patrisiakaplun** [comment]
@idosinger98 Not sure I understand what you mean

**Review by OfirBachar — APPROVED** [review]
LGTM

**Comment by patrisiakaplun** [comment]
@idosinger98 Done

**Review by Yarboa — COMMENTED** [review]
Is there a test to verify all_chats? 
@patrisiakaplun

**Comment by patrisiakaplun** [comment]
@Yarboa Done.

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**